specifications/api/swagger.yaml

```diff
@@ -687,6 +687,33 @@ x-transactionMetadata: &transactionMetadata
     3: { "list": [ { "int": 14 }, { "int": 42 }, { "string": "1337" } ] }
     4: { "map": [{ "k": { "string": "key" }, "v": { "string": "value" }}, { "k": { "int": 14 }, "v": { "int": 42 } }] }
 
+x-transactionTTL: &transactionTTL
+  description: |
+    The TTL (time to live) is the time period in which the transaction
+    will be accepted into node mempools.
+
+    After the TTL has lapsed, the transaction is considered
```

Comment:
genuine English question: when would you favor "lapsed" over "elapsed" ?

Reply:
Lapsed is for a time period which is "over" at the end, and elapsed is for a time period that continues to pass.
But they tend to be used interchangeably.